sdv/tabular/copulas.py

```diff
@@ -289,9 +289,11 @@ def _fit(self, table_data):
                 Data to be fitted.
         """
         for column in table_data.columns:
-            distribution = self._field_distributions.get(column)
-            if not distribution:
-                self._field_distributions[column] = self._default_distribution
+            if column not in self._field_distributions:
+                # Check if the column is a derived column.
+                column_name = column.replace('.value', '').replace('.is_null', '')
```

Comment:
The `is_null` columns are just boolean columns that say whether or not their row should be null. I would apply a fixed distribution to them.

I think it only makes sense to apply the specified distribution to the one ending in `.value`